### PR TITLE
fix: SearchFilterForm mobile view

### PR DESCRIPTION
**Pull request by Mdjuealhossain**

### CODE REVIEW

File: src/components/Logo/index.tsx [hunk 1]
```diff
@@ -43,7 +40,7 @@ const Logo: FC<LogoProps> = () => {
         }}
       >
         DREAM HOUSE
```
Comment (arifulbgt4) [inline src/components/Logo/index.tsx:42]: Svg icon should placed here

File: src/layouts/AppLayout/index.tsx [hunk 2]
```diff
@@ -12,10 +12,10 @@ const AppLayout: FC<AppLayoutProps> = async ({ children }) => {
   const session = await getServerSession(authOptions);
   return (
     <Grid container>
-      <Grid item xs={12}>
+      <Grid item xs={12} height={56.7}>
         <Header user={session?.user} />
       </Grid>
-      <Grid item xs={12}>
+      <Grid item xs={12} height="calc(100% - 56.7px)">
```
Comment (arifulbgt4) [inline src/layouts/AppLayout/index.tsx:18]: Move to variable the header height

**Review by arifulbgt4 — CHANGES_REQUESTED** [review]

**Comment by Mdjuealhossain** [comment]
Screenshot_11
Screenshot_10

File: src/layouts/AppLayout/index.tsx [hunk 3]
```diff
@@ -9,10 +9,12 @@ import Footer from "src/widgets/Footer";
 import { AppLayoutProps } from "./Types";
 
 const AppLayout: FC<AppLayoutProps> = async ({ children }) => {
+  const headerHeight = 64;
```
Comment (arifulbgt4) [inline src/layouts/AppLayout/index.tsx:12]: remove header height

**Review by arifulbgt4 — CHANGES_REQUESTED** [review]

File: src/widgets/ListSearchFilters/Types.ts [hunk 4]
```diff
@@ -1 +1,3 @@
-export interface ListSearchFiltersProps {}
+export interface ListSearchFiltersProps {
+  onClose?: () => void;
```
Comment (arifulbgt4) [inline src/widgets/ListSearchFilters/Types.ts:2]: onCloseMobileDrawer

File: src/widgets/SearchListingGroup/index.tsx [hunk 5]
```diff
@@ -13,11 +13,7 @@ import { SearchListingGroupProps } from "./Types";
 const SearchListingGroup: FC<SearchListingGroupProps> = () => {
   const [isGrid, setIsGrid] = useState(true);
 
-  const handleGrid = (event: boolean) => {
-    setIsGrid(event);
-  };
-
-  const handleList = (event: boolean) => {
+  const handleGridList = (event: boolean) => {
```
Comment (arifulbgt4) [inline src/widgets/SearchListingGroup/index.tsx:16]: toggleGridList

File: src/widgets/SearchListingGroup/index.tsx [hunk 6]
```diff
@@ -13,11 +13,7 @@ import { SearchListingGroupProps } from "./Types";
...
     setIsGrid(event);
```
Comment (arifulbgt4) [inline src/widgets/SearchListingGroup/index.tsx:17]: setIsG(!isGrid)

**Review by arifulbgt4 — CHANGES_REQUESTED** [review]

**Review by arifulbgt4 — APPROVED** [review]

**Pull request merged**